src/libsyntax/json.rs

```diff
@@ -137,7 +131,6 @@ struct DiagnosticSpan {
     /// that should be sliced in atop this span.
     suggested_replacement: Option<String>,
     /// If the suggestion is approximate
-    #[rustc_serialize_exclude_null]
     suggestion_applicability: Option<Applicability>,
```

Comment:
Am I reading the code below correctly as: `suggestion_applicability` is always `Some` when `suggested_replacement` is `Some`? If so, we might want to document this here.